.travis.yml

```diff
@@ -1,23 +1,40 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
+matrix:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 5.6
+      env: DEPENDENCY_INJECTION_VERSION=2.2.* EXPRESSION_LANGUAGE_VERSION=2.4.*
```

Comment:
@matthiasnoback Sure, this pretty much makes sense. I also removed the ExpressionLanguage component for `2.3`.